types/react-redux/react-redux-tests.tsx

```diff
@@ -1388,7 +1388,8 @@ function testUseStore() {
 
     const untypedStore = useStore();
     const state = untypedStore.getState();
-    state.things.stuff.anything; // any by default
+    // untyped store has {} as the state by default
```

Comment:
this test change shows that this is a breaking change. please ensure this test still works without any changes to it